.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to meetupr
+
+This outlines how to propose a change to meetupr. For more detailed
+info about contributing to this, and other tidyverse packages, please see the
+[**development contributing guide**](https://rstd.io/tidy-contrib).
+
+### Fixing typos
+
+Small typos or grammatical errors in documentation may be edited directly using
+the GitHub web interface, so long as the changes are made in the _source_ file.
+
+*  YES: you edit a roxygen comment in a `.R` file below `R/`.
+*  NO: you edit an `.Rd` file below `man/`.
+
+### Prerequisites
+
+Before you make a substantial pull request, you should always file an issue and
+make sure someone from the team agrees that it’s a problem. If you’ve found a
+bug, create an associated issue and illustrate the bug with a minimal 
+[reprex](https://www.tidyverse.org/help/#reprex).
+
+### Pull request process
+
+*  We recommend that you create a Git branch for each pull request (PR).  
+*  Look at the Travis and AppVeyor build status before and after making changes.
```

Comment:
Are we using Appveyor? I saw the Travis badge...

Reply:
We are not. Good catch! I will change that.